CorruptOSBot.Data/CorruptOSBot.Data.csproj

```diff
@@ -10,11 +10,12 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>CorruptOSBot.Data</RootNamespace>
     <AssemblyName>CorruptOSBot.Data</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
Possibly a breaking change when deploying to the server but we should update the server to the latest version anyway